lib/manager.js

```diff
@@ -381,10 +380,18 @@ module.exports = class Manager {
         });
     }
 
-    _render(compiled, context, request) {
+
+    async _render(compiled, context, request, callback) {
 
         if (this._context) {
-            let base = typeof this._context === 'function' ? this._context(request) : this._context;
```

Comment:
This could've probly been kept as a one-liner but this is fine

Reply:
U know, looking back on this comment I feel readability is much better so NVM what I said